spec/active_record/connection_adapters/makara_mysql2_adapter_spec.rb

```diff
@@ -3,10 +3,10 @@
 
 describe 'MakaraMysql2Adapter' do
 
-  let(:db_username){ ENV['TRAVIS'] ? 'travis' : 'root' }
+  let(:db_username){ ENV['TRAVIS'] ? 'travis' : ENV['MYSQL_USER'] || 'root' }
```

Comment:
Oh, I see. yeah that is complicated logic. I guess it depends if travis runs with MYSQL_USER set or not.